src/pageCtrl.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+let app = angular.module('app',[]);
+app.controller('pageController', function($scope) {
+    let details = {};
+    $scope.documents = [];
+    let read = new Index();
+    $scope.fileUpload = function() {
+        let fileDetails = document.getElementById('file').files[0];
+        if(fileDetails===undefined){
+            console.log('No file chosen');
```

Comment:
Good job handling edge cases. It would be nicer if you notify the user about failures instead of just logging them to the console.